src/app/meeting/[id]/page.tsx

```
@@ -0,0 +1,271 @@
+
+  return (
+    <RequireMember>
+      <main className="min-h-screen pb-16 bg-white">
+        <div className="container-x-lg">
+          <div className="pt-6 lg:pt-12">
+            {/* 상단 바 — 목록으로 / 더보기 */}
+            <div className="flex items-center justify-between mb-6">
+              <button
+                onClick={() => router.push("/meeting")}
+                className="inline-flex items-center gap-1 rounded-full border border-gray-200 px-5 py-2.5 text-sm text-gray-600 transition-colors hover:bg-gray-50"
+              >
```

Biome (2.5.0)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)